models/articles.model.js

```diff
@@ -35,5 +35,27 @@ async function selectArticles(sort_by='created_at') {
     }
 }
 
+async function selectCommentsByArticleId(id) {
+    try {
+        const comments = await db.query(`
+        SELECT * FROM comments WHERE article_id=$1 ORDER BY created_at DESC;`, [id]);
+        return comments.rows;
+    } catch (error) {
+        // throw error
+    }
+}
+
+async function checkArticleIdExists(id) {
```

Comment:
This is a nice way of doing it, but your pre-written fetchArticleById function should also work